test/test.py

```diff
@@ -728,6 +728,23 @@ def assert_bundle():
 
         assert_bundle()
 
+    def test_name_substitution(self):
+        uuid = 'ffee7f29-5c38-461a-8771-a68e20ec4a2e'
+        version = '2019-02-02T065454.662896Z'
+        manifest, metadata_files = self._load_bundle(uuid, version, replica='aws', deployment='prod')
+        files_with_bang = [f['name'] for f in manifest if '!' in f['name']]
+        self.assertNotEqual(len(files_with_bang), 0)
```

Comment:
assertGreater but see below

Reply:
Test was changed to assert contents verbatim.